src/main/java/org/prebid/server/bidder/compass/proto/ImpExtCompass.java

```diff
@@ -0,0 +1,17 @@
+package org.prebid.server.bidder.compass.proto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+
+@Builder
+public class ImpExtCompass {
```

Comment:
Nitpick: rename to `CompassImpExtBidder`

Reply:
Maybe better rename to just `CompassImpExt` because we use builder and it'll be name `CompassImpExtBidder.CompassImpExtBidderBuilder` for just create .

Reply:
If it seems to long, you can static import `CompassImpExtBidderBuilder `